eng/scripts/Pack-Nuget.ps1

```diff
@@ -51,6 +50,7 @@ try {
 		$serverProjectProperties = & "$projectPropertiesScript" -ProjectName "$serverName.csproj"
 		$platformOutputPath = "$OutputPath/nuget/$($serverDirectory.Name)/platform"
 		$wrapperOutputPath = "$OutputPath/nuget/$($serverDirectory.Name)/wrapper"
+		$packageIconPath = if ($serverProjectProperties.PackageId.StartsWith("Azure.")) {"$RepoRoot/eng/images/azureicon.png"} else {"$RepoRoot/eng/images/microsofticon.png"}
```

Comment:
I would just add `<PackageIconPath>$(RepoRoot)/eng/images/azureicon.png</PackageIconPath>` to Azure.Mcp.Server.csproj and then default it to the Microsoft icon here if the project doesn't override.

```ps
$packageIconPath = $serverProjectProperties.PackageIconPath
if (!$packageIconPath) {
  $packageIconPath = "$RepoRoot/eng/images/microsofticon.png"
}
```

Reply:
i.e.  nothing in `eng/` should special case any specific server